index.html

```diff
@@ -1,122 +1,901 @@
 <!doctype html>
 <html lang=en>
+
 <head>
-<meta charset=utf-8>
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<title>Congratulations | Cloud Run</title>
-<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-<link
-  href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAACXBIWXMAAAsSAAALEgHS3X78AAAB5klEQVRYha2XMVLDMBBFPxl6qCjciBskNzA3AE5AqrTQuaVVR1pX4QSYGyQ3SGqGATcu3JBUlGGUiTPyeiWvZH6TsSTLb/+XN8nZfr+H0lUBYAvgucySbzBSupoCmJRZ8sTNx2p0vG8J4AHAl9LVQunqmtnPQD4qXa0d81GyARo1IC/2hmWWGIfeAYwBGIi7fwMos2QNYEfmTLWXZKw4fl4AeKOQQxwAcaERrbIg14MjCQKwYrA1KJI+gBtmjLqAIZEcXsNGSlfb42a27sssKaw15lz8ePbcGOdcrzPViFz3uuCIwVZQJBIAbiMuBlviSCQASulqEgjQqPctaQE4+gEiYrDljYQ6AIcLU2ZM6gJ8kUgBxp6uGKJOJFIACJuSRK1IOgCecxDzNrh0iuScW/D7+eG4LxGuk6sDcDVbTZhuCEe1sV/JxuFpnacF5wDX/zsAV7OVOZS3EQ8/tOo6Tw+tWgqwqfN0S8Ziqp/Xedr6SScFWDBjIQAny+lEC8CT/5KsC7G/ZbkXwFF9WefpmoxJq+9YHgMQc/qdlscAhNrvtdwJ4Mh/x1Thq77XcieApHoPgNjyUABJ8wmyfBAAU32w5SyAI/+5p/tFW84CkOpfzb9kaqll/yDLfQDsgy2Z6gdb3hKAP2n+6m+fxBVHAAAAAElFTkSuQmCC"
-  rel="icon"
-  type="image/png" 
-/>
-<style>
-  body {
-    font-family: 'Roboto', sans-serif;
-  }
-  .container {
-    max-width: 800px;
-    margin-left: auto;
-    margin-right: auto;
-  }
-  .hero {
-    margin-top: 100px;
-    margin-bottom: 100px;
-  }
-  .hero .logo {
-    width: 140px;
-    float: left;
-    margin-right: 32px;
-  }
-  .hero .message {
-  }
-  .details {
-    clear: both;
-  }
-  .warning {
-    color: rgb(213, 0, 0);
-    background: #fbe9e7;
-    margin-bottom: 1.0em;
-    padding: 1.0em;
-  }
-
-  .warning a:visited {
-    color: rgb(213, 0, 0);
-  }
-
-  .warning a:link {
-    color: rgb(213, 0, 0);
-  }
-
-  .cta {
-    text-align: center;
-    margin: 32px;
-  }
-  .cta a {
-    color: white;
-    text-decoration: none;
-    background-color: rgb(66, 133, 244);
-    padding: 18px 24px;
-    margin: 12px;
-    min-height: 36px;
-    border-radius: 4px;
-    font-weight: 600;
-  }
-  .cta a:hover {
-    background-color: rgb(66, 133, 244, 0.9);
-  }
-  @media (max-width: 600px) {
-    .hero .logo {
-      float: initial;
-      display: block;
-      margin-left: auto;
-      margin-right: auto;
-    }
-    .container {
-      margin-top: 0;
-    }
-    .hero {
-      margin-top: 0;
-      margin-bottom: 24px;
-    }
-  }
-</style>
+	<meta charset=utf-8>
```

Comment:
Could you keep the original indentation. Many of these lines have not changed.

Reply:
Yes.